### ISSUE TITLE
Hydration Pack Volume Incorrect

### ISSUE THREAD

**Issue by ExplosiveRunes** [CleverRaven/Cataclysm-DDA#73508]
### Describe the bug

Recently the hydration pack's volume changed from .01 L empty to 3.00 L empty, and expands to 6.00L when full. This prevents it from being placed within pockets specifically designed for it when full, such as the "pouch sized for a hydration pack." on the MOLLE assault pack. The Hydration Pack may not be the only item affected by whatever has changed, but it is the item that I noticed this on.

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/5423263/d42cd995-9c87-4a44-b2a0-e93fbfcea48c)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/5423263/f836ff20-d158-4c8b-85e1-b5a90a6ef3e8)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/5423263/2c59f797-bc68-423f-9e2e-36a625083e8a)

### Attach save file

N/A

### Steps to reproduce

1. Create new world with only Dark Days Ahead content pack
2. Spawn in Hydration pack
3. Observe volume when filled an unfilled

### Expected behavior

The hydration pack previously had a volume of .01 L empty and would expand to include the volume of the contained liquid up to its maximum.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22631.3155 (23H2)
- Game Version: 00d3e82 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda]
]

**Comment by Hyperseeker** [comment]
Caused by https://github.com/CleverRaven/Cataclysm-DDA/pull/73458/. Hydration pack used to be 3001 mL in volume, with its `rigid` bladder holding up to 3000 mL (one millilter less) of liquid.

The fix is simple, but not quite. In real life, neither any of the Camelbak family of hydration bladders nor the MOLLE II hydration bladder (which is relevant because military backpacks in the game have a pocket for it) is only 1 mL in volume. From my calculations for [Armory](https://github.com/FrontierMods/Armory), the MOLLE II bladder is just over 250 mL in volume, empty. This would make a full 3 L bladder (which it is rated for) just over 3250 mL. This could be trimmed down to an actual 3250 mL, for the purpose of the game being nice and even with its values.

Basically, whoever takes up fixing this should just make the hydration pack ~250 mL in volume, and adjust the pockets that it's supposed to sit in (MOLLE packs etc.) accordingly.

(Worth noting that `camelbak`, the in-game item, is supposed to be two items in one: the bladder which holds the liquids, and the thin sleeve of a backpack that would allow carrying the bladder on their person. Right now, you can carry the bladder on your shoulders like a regular backpack. The MOLLE II carrier – i.e., said backpack – could add anywhere between 500 mL and 1500 mL on its own, depending on the thickness of the nylon.)
